### PR TITLE
removed alpha chain oracle for devrel node, with ap election data

### PR DESCRIPTION
**Pull request by PatrickAlphaC**
Fixes: https://github.com/smartcontractkit/documentation/issues/447

### CODE REVIEW

File: docs/Using Any API/existing-job-request.md [hunk 1]
```diff
@@ -11,27 +11,29 @@ metadata:
...
-# OpenWeather Consumer
+# AP Election Data Consumer
 
 In [Make a GET Request](../make-a-http-get-request/), the example contract code declared which URL to use, where to find the data in the response, and how to convert it so that it can be represented on-chain.
 
-In this example, we're using a job found on the <a href="https://market.link/" target="_blank">Chainlink Market</a> that is pre-configured to perform these tasks. This means that our contract doesn't need to specify additional parameters for various adapters, it only needs the Oracle address and the Job ID. The remaining adapters are configured by the external adapter, in particular <a href="https://market.link/adapters/5ff8f621-102d-491d-b1c8-bbbe294e4620" target="_blank">weather_cl_ea</a>.
+This example uses an existing job that is pre-configured to make requests to get [AP Election data](https://developer.ap.org/ap-elections-api/). 
+
+> 📘 Job Spec
+> The job spec for the Chainlink node in this example can be [found here](https://github.com/smartcontractkit/documentation/_includes/samples/APIRequests/ap-election-job.md).
```
Comment (dwightjl) [inline docs/Using Any API/existing-job-request.md:23]:
```suggestion
You can find the job spec for this example on [GitHub](https://github.com/smartcontractkit/documentation/tree/main/_includes/samples/APIRequests/ap-election-job.toml).
```

**Review by dwightjl — CHANGES_REQUESTED** [review]

**Comment by dwightjl** [comment]
This seems to work: https://kovan.etherscan.io/tx/0x59abeb9c7531c5214e1ed147b1de7a4dbf4fa7819a2584e9ce4e0f958ce74554#eventlog

However Remix isn't displaying any results after I run `requestElectionWinner()`. Only `0x0000...` values. I don't have an API key so I can't check the AP Elections API directly.

**Comment by PatrickAlphaC** [comment]
You shouldn't need to use an API key... The job I gave has the data defined on the Chainlink node itself. So you should just be able to run `APElection.sol ` and be good. Are you using an AP URL yourself?

File: _includes/samples/APIRequests/APElection.sol [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    /**
+     * Initial request
+     */
+    function requestElectionWinner() public returns (bytes32 requestId) 
+    {
+        Chainlink.Request memory request = buildChainlinkRequest(jobId, address(this), this.fulfill.selector);
+        request.add("date", "2021-11-02");
+        request.add("raceId", "12111");
+        request.add("statePostal", "FL");
+        request.add("resultsType", "t"); // Replace with l for live results.
```
Comment (dwightjl) [inline _includes/samples/APIRequests/APElection.sol:58]:
```suggestion
        request.add("resultsType", "l"); // Replace with l for live results.  
```
Change to live results.

**Review by dwightjl — CHANGES_REQUESTED** [review]

**Review by dwightjl — APPROVED** [review]

**Pull request merged**